File: ntss-win/P2045ReportApp/LayoutDesigner/Data/SysReportClass.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LayoutDesigner.Data
{
    /// <summary>
    /// 帳票種別定義
    /// </summary>
    [System.Runtime.Serialization.DataContract()]
    class SysReportClass
    {
        /// <summary>
        /// 帳票種別
        /// </summary>
        [System.Runtime.Serialization.DataMember(Name = "reportClassCd")]
        public string ReportClassCd { get; set; }

        /// <summary>
        /// 帳票種別名
        /// </summary>
        [System.Runtime.Serialization.DataMember(Name = "reportClassName")]
        public string ReportClassName { get; set; }

        /// <summary>
        /// 帳票区分
        /// </summary>
        [System.Runtime.Serialization.DataMember(Name = "reportType")]
        public string ReportType { get; set; }

        /// <summary>
        /// 表示フラグ
        /// </summary>
        [System.Runtime.Serialization.DataMember(Name = "isDisp")]
        public string IsDisp { get; set; }

        /// <summary>
        /// 削除フラグ
        /// </summary>
        [System.Runtime.Serialization.DataMember(Name = "isDel")]
        public string IsDel { get; set; }

        /// <summary>
        /// 更新日時
        /// </summary>
        [System.Runtime.Serialization.DataMember(Name = "regDate")]
        public string UpDate { get; set; }

        /// <summary>
        /// 登録日時
        /// </summary>
        [System.Runtime.Serialization.DataMember(Name = "upDate")]
        public string RegDate { get; set; }
    }
}

File: Source/ntss-src/ntss-core/src/main/java/jp/co/nikkiso/ntss/core/entity/SysReportClass.java
package jp.co.nikkiso.ntss.core.entity;

import lombok.Getter;
import lombok.Setter;
import org.seasar.doma.Entity;
import org.seasar.doma.Id;
import org.seasar.doma.Table;
import org.seasar.doma.jdbc.entity.NamingType;

import java.sql.Timestamp;

/**
 *    * 帳票種別定義サービス.
 */
@Entity(naming = NamingType.SNAKE_LOWER_CASE)
@Table(name = "sys_report_class")
@Getter
@Setter
public class SysReportClass extends BaseBlankEntity {

  /**
   * 種別コード
   */
  @Id
  private String reportClassCd;

  /**
   * 種別名
   */
  private String reportClassName;

  private String reportType;


  /**
   * 表示フラグ
   */
  private String isDisp;

  /**
   * 削除フラグ
   */
  private String isDel;

  /**
   * 登録日時
   */
  private Timestamp regDate;

  /**
   * 更新日時
   */
  private Timestamp upDate;

}
